p2p/peer.py

```diff
@@ -615,6 +647,10 @@ class ETHPeer(BasePeer):
         self.head_td = msg['td']
         self.head_hash = msg['best_hash']
 
+    def request_block_headers(self, block_number_or_hash: Union[int, bytes],
+                              max_headers: int, reverse: bool = True) -> None:
+        self.sub_proto.send_get_block_headers(block_number_or_hash, max_headers, reverse)
```

Comment:
It took me a little digging to convince my self that it wasn't a bug that these methods named `send_get_block_headers` have different positional arguments. Maybe rename one of them?

Reply:
Well, both sub-protocols (LES and ETH) use the same msg name (GetBlockHeaders), but the msg format itself is different and that's why the method signatures are different. I'd rather keep the method names as they are -- that is, just prefixing the msg name with `send_`